ogusa/income.py

```diff
@@ -188,12 +188,48 @@ def get_e_interp(S, age_wgts, age_wgts_80, abil_wgts, plot=False):
     # Get original 80 x 7 ability matrix
     abil_wgts_orig = np.array([0.25, 0.25, 0.2, 0.1, 0.1, 0.09, 0.01])
     emat_orig = get_e_orig(age_wgts_80, abil_wgts_orig, plot)
-
-    # Return emat_orig if S = 80 and abil_wgts = abil_wgts_orig
-    if S == 80 and np.array_equal(abil_wgts,
-                                  np.array([0.25, 0.25, 0.2, 0.1, 0.1,
-                                            0.09, 0.01])) is True:
+    if (S == 80 and np.array_equal(
```

Comment:
@jdebacker. At the end of line 168 of this `get_e_interp()` function docstring, we should include some language like the following:
> ...too much curvature.] This function also includes the two cases in which J = 9 and J = 10 that include Piketty and Saez higher lifetime earning percentiles.

This way, the changes to this function show up in the API documentation.